Applications/DataExplorer/VtkVis/VtkMeshSource.cpp

```diff
@@ -144,39 +145,40 @@ int VtkMeshSource::RequestData( vtkInformation* request,
 		for (unsigned j = 0; j < nElemNodes; ++j)
 			point_ids->SetId(j, elem->getNode(j)->getID());
 
-		switch (elem->getGeomType())
+		type = OGSToVtkCellType(elem->getCellType());
+		if (type==0)
```

Comment:
A constant 0 could potentially change. Better to use something along
```c++
if (type == CellType::INVALID)
```
(Introduce it if necessary).